seqlog/structured_logging.py

```diff
@@ -427,3 +435,44 @@ def _get_local_timestamp(record):
     )
 
     return timestamp.isoformat(sep=' ')
+
+
+def _ensure_class(class_or_class_name, compatible_class=None):
+    """
+    Ensure that the supplied value is either a class or a fully-qualified class name.
+
+    :param name: A class or a fully-qualified class name.
+    :param expected_base_class: If specified then the class must be, or (eventually) derive from, this class.
+    :return: The class represented by class_or_class_name.
+    """
+
+    target_class = class_or_class_name
+    if isinstance(class_or_class_name, str):
+        name_parts = class_or_class_name.split('.')
+        module_name = '.'.join(
+            name_parts[:-1]
+        )
+        target_class_name = name_parts[-1:]
```

Comment:
`target_class_name` will be a list, should be :
`target_class_name = name_parts[-1]`

Reply:
Oops! Yes, sorry I'll fix that up and merge this morning!